packages/pavuk.rb

```diff
@@ -0,0 +1,24 @@
+require 'package'
+
+class Pavuk < Package
+  description 'Web spider and website mirroring tool. http://www.pavuk.org/'
+  homepage 'https://sourceforge.net/projects/pavuk/files/pavuk/'
```

Comment:
Please do not include the homepage url in the description.  If you want to propose that we add a new field to the Package class, then please create a new issue so we can have a discussion.  The homepage url you have there also points at the source, not the homepage of the project.

Reply:
I don't want to 'spend' 'more time' on talking, I will contribute, but remove one link(in the section, the direct link to download). Do, what you want with rest I'm not interested(don't take it personally, it is not).